art/estimators/object_detection/PyTorchFasterRCNN.py

```diff
@@ -160,6 +180,8 @@ def loss_gradient(self, x, y, **kwargs):
 
         assert grads.shape == x.shape
 
+        grads = grads / self.clip_values[1]
```

Comment:
See the previous comment.

Reply:
You are right. The reason for this is that this estimator is model-specific for PyTorch's FasterRCNN which expects images in the pixel range [0, 1]. To enable inputs to `PyTorchFasterRCNN` in the range [0, 255] it normalises with `clip_values[1]`. I have now added assertions to prevent unexpected `clip_values` where where `min_clip!=0`.